helios-service-registration/src/main/java/com/spotify/helios/serviceregistration/ServiceRegistration.java

```diff
@@ -24,6 +24,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import com.google.common.base.Objects;
```

Comment:
Really nitpicking here but I don't think this `import` should be in the same stanza as the static imports.